### PR TITLE
Change ign with gz

### PR DESCRIPTION
**Pull request by marcoshuck**
Due to the migration to GitHub, we also renamed the package to `gz-go`. It makes sense for it to be called `gz` instead of `ign`. This PR changes the package definition to `gz`.

This was tested on a testing branch in a private repository and it works as expected.

- CI will be introduced in a future PR.
- We haven't changed the env vars prefix from `IGN_` to `GZ_`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,32 +18,32 @@ Ignition Go provides a set of features to help with web server development. It i
```
Comment (AlejoAsd) [inline README.md:18]: Github doesn't allow me to comment on things that were not changed in a PR, but the intro before this line here is using Ignition in a number of places.

File: aws_test.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package ign
+package gz
 
 import (
 	"github.com/stretchr/testify/assert"
```
Comment (AlejoAsd) [inline aws_test.go:4]: The test below this line is using the `ign` prefix.

**Review by AlejoAsd — CHANGES_REQUESTED** [review]
There's a number of files that are still using `ign` or `Ignition` in them:

1. `README.MD`
2. `gorm.go`
3. `logger.go`
4. `main.go` (Mostly due to env vars)
5. `ip_address.go`
6. `router_middleware.go`
7. `router_test_helpers.go`
8. `test_helpers.go`

I won't comment on the old CI files as you already mentioned they will be addressed in another PR.

Let me know if you want to address the rest of the Ignition reference in another PR.

File: README.md (hunk 1 above)
Comment (marcoshuck) [reply README.md:18]: We can probably change documentation in a future PR, wdyt? We can probably add a task to follow up on this.

File: aws_test.go (hunk 2 above)
Comment (marcoshuck) [reply aws_test.go:4]: Good catch, let me take a look at it.

File: README.md (hunk 1 above)
Comment (AlejoAsd) [reply README.md:18]: That's okay with me.

Comment (marcoshuck) [reply README.md:18]: Task added to asana: https://app.asana.com/0/851925973517080/1203590389490573

**Review by AlejoAsd — APPROVED** [review]

**Pull request merged**